packages/starlight/components/SidebarSublist.astro

```diff
@@ -120,8 +128,9 @@ interface Props {
 		background-color: var(--sl-color-text-accent);
 	}
 
-	a > *:not(:last-child) {
-		margin-inline-end: 0.25em;
+	a > *:not(:last-child),
+	summary > div.group-label > *:not(:last-child) {
+		margin-inline-end: 0.5em;
```

Comment:
Do we need to increase this spacing? I think the previous value should work OK as long as we reintroduce the whitespace between text and badge.

```suggestion
		margin-inline-end: 0.25em;
```

Reply:
This was intentional, I couldn't get the whitespace to work using `{" "}`. it's quite unpredictable. I think it's best to rely more in css for these situations.

Reply:
Oh nevermind, it worked this time 😅.